.github/workflows/inter-branch-merge-flow.yml

```
@@ -10,4 +10,4 @@ permissions:
 
 jobs:
   Merge:
-    uses: dotnet/arcade/.github/workflows/inter-branch-merge-base.yml@main
\ No newline at end of file
+    uses: dotnet/arcade/.github/workflows/inter-branch-merge-base.yml@1912d9f4fc410d421a01b5a09131aae234b603fa # main
```

Nit-picky: In the absence of a tag, I recommend grabbing the last commit for this workflow and commenting it with the date of that change rather than an arbitrary SHA from other changes in main.

```suggestion
    uses: dotnet/arcade/.github/workflows/backport-base.yml@2e09b2a8a74223d25086d947e2d070827f6c556e# 2024-06-24
```